searching/linear_search.c

```diff
@@ -1,36 +1,67 @@
-#include <stdio.h>
-#include <stdlib.h>
+/**
+ * @file
+ * @brief Algorithm to Search an Element from the array
```

Comment:
Provide a Wikipedia link in Markdown format (if available/possible).
If there's no Wikipedia link, add another web source for algorithm explanation. 🙂

Reply:
Done..!
Hope now it meets the requirements😊

Thanks !!